tests/garage/tf/policies/test_gaussian_mlp_policy_with_model.py

```diff
@@ -105,57 +100,61 @@ def test_gaussian_mlp_policy_kl_sym(self):
         kl2 = self.sess.run(
             objective2,
             feed_dict={
-                self.policy3.model.networks['default'].input: self.obs,
-                self.policy4.model.networks['default'].input: self.obs
+                self.policy3.model.networks['default'].input: self.zeros_obs,
```

Comment:
hmm shouldn't this be true for *any* input, not just the zero input?

Reply:
It should be. I wanted the data to have zero mean, since the weights of the mean network is different. 
I think one way is to copy the mean network weights of policy1 to policy3, and policy2 to policy4 so the output for mean network would be the same.

Reply:
yes, I will change that.

Reply:
Actually can I remove GaussianMLPPolicyWithModel2 at this moment first? At the first step we keep garage.Distribution first, so we will only keep GaussianMLPPolicyWithModel and make sure it works with existing GaussianMLPPolicy.

Later when we start to replace garage.Distribution with tfp.distribution, I will put GaussianMLPPolicyWithModel2 back.